web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-config-topics.adoc

```diff
@@ -0,0 +1,92 @@
+= Web Service Consumer Additional Configuration Information - Mule 4
+:page-aliases: connectors::web-service/web-service-consumer-configure.adoc
+
+Set additional configuration for Anypoint Connector for Web Service Consumer (Web Service Consumer Connector), such as custom HTTP Transport and Web Service Security both in Anypoint Studio and XML editor.
```

Comment:
"... and in the XML editor."
I think that "Transport" and "Security" should be lowercase.